docs/patronictl.rst

```diff
@@ -0,0 +1,1963 @@
+
+.. _patronictl_resume:
+
+patronictl resume
+^^^^^^^^^^^^^^^^^
+
+.. _patronictl_resume_synopsis:
+
+Synopsis
+""""""""
+
+.. code:: text
+
+    resume
+      [ CLUSTER_NAME ]
+      [ --group CITUS_GROUP ]
+      [ --wait ]
+
+.. _patronictl_resume_description:
+
+Description
+"""""""""""
+
+``patronictl resume`` takes the Patroni cluster out of maintenance mode and re-enables automatic failover.
+
+.. _patronictl_resume_parameters:
+
+Parameters
+""""""""""
+
+``CLUSTER_NAME``
+    Name of the Patroni cluster.
+
+    If not given, ``patronictl`` will attempt to fetch that from the ``scope`` configuration, if it exists.
+
+``--group``
+    Resume the given Citus group.
+
+    ``CITUS_GROUP`` is the ID of the Citus group.
+
+``--wait``
+    Wait until all Patroni members are unpaused before returning control to the caller.
+
+.. _patronictl_resume_examples:
+
+Examples
+""""""""
+
+Put the cluster out of maintenance mode:
+
+.. code:: text
+
+    patronictl -c postgres0.yml resume batman --wait
+    'resume' request sent, waiting until it is recognized by all nodes
+    Success: cluster management is resumed
+
+.. _patronictl_show_config:
+
+patronictl show-config
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. _patronictl_show_config_synopsis:
+
+Synopsis
+""""""""
+
+.. code:: text
+
+    show-config
+      [ CLUSTER_NAME ]
+      [ --group CITUS_GROUP ]
+
+.. _patronictl_show_config_description:
+
+Description
+"""""""""""
+
+``patronictl show-config`` shows the dynamic configuration of the cluster that is stored in the DCS.
+
+.. _patronictl_show_config_parameters:
+
+Parameters
+""""""""""
+
+``CLUSTER_NAME``
+    Name of the Patroni cluster.
+
+    If not given, ``patronictl`` will attempt to fetch that from the ``scope`` configuration, if it exists.
+
+``--group``
+    Show dynamic configuration of the given Citus group.
+
+    ``CITUS_GROUP`` is the ID of the Citus group.
+
+.. _patronictl_show_config_examples:
+
+Examples
+""""""""
+
+Show dynamic configuration of cluster ``batman``:
+
+.. code:: text
+
+    patronictl -c postgres0.yml show-config batman
+    loop_wait: 10
+    postgresql:
+      parameters:
+        max_connections: 250
+      pg_hba:
+      - host replication replicator 127.0.0.1/32 md5
+      - host all all 0.0.0.0/0 md5
+      use_pg_rewind: true
+    retry_timeout: 10
+    ttl: 30
+
+.. _patronictl_switchover:
+
+patronictl switchover
+^^^^^^^^^^^^^^^^^^^^^
+
+.. _patronictl_switchover_synopsis:
+
+Synopsis
+""""""""
+
+.. code:: text
+
+    switchover
+      [ CLUSTER_NAME ]
+      [ --group CITUS_GROUP ]
+      [ { --leader | --primary | --master } LEADER_NAME ]
+      --candidate CANDIDATE_NAME
+      [ --force ]
+
+.. _patronictl_switchover_description:
+
+Description
+"""""""""""
+
+``patronictl switchover`` performs a switchover in the cluster.
+
+It is designed to be used when the cluster is healthy, e.g.:
+
+- There is a leader; or
+- There are synchronous standbys available in a synchronous cluster.
```

Comment:
I read it as `we could have synchronous standbys but no leader`, which is wrong.

Reply:
Removed the "or". I guess that's enough, right?